app/dts/behaviors/caps_word.dtsi

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <dt-bindings/zmk/keys.h>
+
+/ {
+	behaviors {
+		/omit-if-no-ref/ caps_word: behavior_caps_word {
+			compatible = "zmk,behavior-caps-word";
+			label = "CAPS_WORD";
+			#binding-cells = <0>;
+			breaks = <SPACE COMMA DOT>;
```

Comment:
Is this default list of space, comma, and dot reasonable? Others we should add?

Reply:
colon, semicolon, equals, line break, tab, ...
Maybe its easier to make it a whitelist? It would also address your above problem. So just note down the keycodes which should get the modifier applied until something else gets typed. I think this list will be always shorter as I barely see what else except alpha keycodes and underscore, dash that should be used here to write a caps word.

Reply:
> Is this default list of space, comma, and dot reasonable? Others we should add?

Escape is a must for vim users!

Reply:
I think ENTER and TAB should also be in the default list